ui/packages/catalog-ui-search/src/main/webapp/component/query-advanced/query-advanced.view.js

```diff
@@ -145,13 +179,15 @@ module.exports = Marionette.LayoutView.extend({
     }
   },
   cancel: function() {
+    fetchFacets.cache.clear()
     this.$el.removeClass('is-editing')
     this.onBeforeShow()
     if (typeof this.options.onCancel === 'function') {
       this.options.onCancel()
     }
   },
   save: function() {
+    fetchFacets.cache.clear()
```

Comment:
❓ Maybe I'm misunderstanding what's being done here, but why would you clear the cache when the query is saved?